src/addons/dragAndDrop/backgroundWrapper.js

```diff
@@ -107,11 +111,15 @@ function createWrapper(type) {
     drop(_, monitor, { props, context }) {
       const event = monitor.getItem();
       const { value } = props
-      const { onEventDrop } = context
+      const { onEventDrop, startAccessor, endAccessor } = context
 
       onEventDrop({
         event,
-        ...getEventTimes(event, value, type)
+        ...getEventTimes({
+          ...event,
+          start: get(event, startAccessor),
+          end: get(event, endAccessor)
```

Comment:
`getEventTimes()` was only looking at the `start`/`end` time so spreading `...event` wasn't strictly necessary. Only did it to minimize the change.